src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form"
+    method="get"
+    action="/"
+    >
+      <div class="container">
+        <label for="search-input" data-qa="keypress">
+          <img
+          class="image"
+          src="images/Search.svg"
+          alt="none"
+          >
+        </label>
+        <input
+        class="input"
+        data-qa="big"
+        type="search"
+        name="search"
+        id="search-input"
+        placeholder="Try &#34Los Angeles&#34"
+        >
+      </div>
+    </form>
+    <form
+    class="search-form small"
+    method="get"
+    action="/">
+      <div class="container">
+        <label for="search-input2">
+          <img
+          class="image image--small"
+          src="images/Search.svg"
+          alt="none"
+          >
```

Comment:
```suggestion
          <img
            class="image image--small"
            src="images/Search.svg"
            alt="none"
          >
```
![image](https://user-images.githubusercontent.com/96409563/202134854-a7ef0448-3952-4801-be36-8321832cc093.png)
Please fix all cases